conf/output/test-hebis-to-lobid-output-1.json

```diff
@@ -0,0 +1,76 @@
+{
+  "@context" : "http://lobid.org/resources/context.jsonld",
+  "id" : "http://lobid.org/resources/512839662#!",
+  "type" : [ "BibliographicResource", "Book" ],
+  "medium" : [ {
+    "label" : "Print",
+    "id" : "http://rdaregistry.info/termList/RDAproductionMethod/1010"
+  } ],
+  "title" : "Alter Dom St. Johannis",
+  "almaMmsId" : "512839662",
+  "oclcNumber" : [ "1407066536" ],
+  "otherTitleInformation" : [ "1500 Jahre Geschichte im Herzen von Mainz" ],
+  "publication" : [ {
+    "startDate" : "2023",
+    "type" : [ "PublicationEvent" ],
+    "location" : [ "Mainz" ],
+    "publishedBy" : [ "Evangelisches Dekanat Mainz" ]
+  } ],
+  "describedBy" : {
+    "resultOf" : {
+      "object" : {
+        "id" : "http://sru.hebis.de/sru/DB=2.1?query=pica.ppn+%3D+%22512839662%22&version=1.1&operation=searchRetrieve&stylesheet=http%3A%2F%2Fsru.hebis.de%2Fsru%2F%3Fxsl%3DsearchRetrieveResponse&recordSchema=marc21&maximumRecords=10&startRecord=1&recordPacking=xml&sortKeys=LST_Y%2Cpica%2C0%2C%2C"
+      }
+    }
+  },
+  "sameAs" : [ {
+    "id" : "http://worldcat.org/oclc/1407066536",
+    "label" : "OCLC Ressource"
+  } ],
+  "related" : [ {
+    "note" : [ "Parallele Sprachausgabe englisch" ],
+    "label" : "The ancient cathedral St. John"
+  } ],
+  "language" : [ {
+    "id" : "http://id.loc.gov/vocabulary/iso639-2/ger",
+    "label" : "Deutsch"
+  } ],
+  "extent" : "1 gefaltetes Blatt ( 16 Seiten) : Illustrationen",
+  "hasItem" : [ {
+    "label" : "lobid Bestandsressource",
+    "type" : [ "Item", "PhysicalObject" ],
+    "callNumber" : "Mog m 7188"
+  }, {
+    "heldBy" : {
```

Comment:
The structure here seems to be wrong: `heldBy` is in a separate object, but should be in the object above?

Reply:
Fixed this.